open-metadata-implementation/user-interfaces/ui-admin-services/docs/user/configuring-ui-server-basic-properties.md

```diff
@@ -0,0 +1,60 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Configuring the basic properties of an OMAG server
+
+The basic properties of the logical OMAG server are used in logging and events originating
+from the server. They help to document the purpose of the server (which helps with problem determination)
+and enable performance improvements by allowing the server to ignore activity or
+metadata that is not relevant to its operation.
+
+The basic properties are:
+
+* **localServerId** - Unique identifier for this server. By default, this is
+  initialized to a randomly generated Universal Unique identifier (UUID).
+
+* **localServerName** - meaningful name for the server for use in messages and UIs. Ideally this value is
+  unique to aid administrators in understanding the source of messages and events from the server.
+  This value is set to the server name assigned when the configuration is created.
+
+* **localServerType** - descriptive type name for the server.  Again this is useful information for the
+  administrator to understand the role of the server. The default value is "Open Metadata and Governance Server".
+ 
+* **organizationName** - descriptive name for the organization that owns the local server/repository.
+  This is useful when the open metadata repository cluster consists of metadata servers from different
+  organizations, or different departments of an enterprise.  The default value is null.
+
+* **localServerURL** - network address of the UI server platform where this server runs
+  (typically host and port number but must also include the initial part of the URL before "open-metadata" if defined).
+  The default value is "http://localhost:8080".
```

Comment:
8443

Reply:
Also other references in this doc to OMAGServer should be UIServer & again port should be 8443 (below)

Reply:
The docs still need correcting for the UI - refers to OMAG server  and also uses examples like cocoMDS1 - should be consistent with 'aaa' or whatever is chosen int he comments above

Reply:
changed